Enhancement: Adds additional content and detailed demonstration to Strapi.md

KarenJoo opened the pull request:
This PR aims to improve the depth and usefulness of the Strapi case study (strapi.md), providing readers with comprehensive guidance with a detailed walkthrough on getting started with Strapi and testing API endpoints using Postman.

**Changes Made:**
- Expanded the _'introduction'_ section to briefly mention the two approaches for using Strapi.
- Updates _'Use case: creating an API with Strapi_': to include the second approach which does not requires integrating Strapi with PostgreSQL.
- Separating the approaches to using Strapi in two sections: _1. Strapi with PostgreSQL_, _2. Strapi using the Quickstart option_ > including testing the endpoints from 2. approach in Postman
- Added a detailed walkthrough guide for setting up Strapi using the Quickstart option and testing the API endpoints in Postman.
- Stored images in a new strapi-folder: src/assets/cms/strapi
- Updates alternative text to all images
- Expanded the _'summary'_ section to include the two different pathways for getting started with strapi
- Updates Credits, Tags and Keywords

sulenchy reviewed (approved):
Great update!!! 
However, The market comparison needs to be included.

KarenJoo commented:
Hey Abi, thanks for feedback. I have done some updates:
- Added a section to the strapi.md with header text "Market Comparison"
- The content in "Market Comparison" is retrieved from an article from Medium.com, comparing the most popular Headless CMS in 2024
- Summarized comparison in a table 
- Adjusted some structure in the "Deploying API" section
